### DISCUSSION TITLE
Config for Renamed Packages?

### DISCUSSION THREAD

**Discussion by straub** [renovatebot/renovate#12500] [Request Help] [answered]
Hey folks,

I was wondering if there's any way currently to configure migrating/updating a package that's been renamed. I saw regexManagers' [autoReplaceStringTemplate](https://docs.renovatebot.com/configuration-options/#autoreplacestringtemplate), and I think that's close. Is there something similar that works for "normal" managers? I looked through the other configuration options and nothing stood out, but always possible I'm missing something obvious.

Ideally, I'd like to be able to tell Renovate that package `prefix-foobar` is now `@prefix/foobar` in my shared config, so the bot can transparently PR a switch to the new name and keep the version updates flowing.

If this has been discussed previously, my search didn't turn it up. Please let me know if this belongs somewhere else!

Thanks!

**Comment by rarkins** [comment, marked as answer]
There is none currently. See #5558

**Reply by straub** [reply]
Ah, "deprecated" was the key word. Thank you!
